config/snapshotgc/snapshotgc.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: snapshot-garbage-collector
+spec:
+  schedule: "* * * * *" # FIXME
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: test-gc
+              image: >-
+                quay.io/redhat-user-workloads/rhtap-integration-tenant/integration-service/integration-service@sha256:7e803efa5aa9561d5869096d3642106d0dae1567831e1ed6f2b0c7ac3e37fdcb
```

Comment:
Probably kustomization in infra deployments should set this into digest, for dev purposes we should keep here `latest`

Reply:
ok. I'll try to figure this out.

Reply:
I corrected the image name and confirmed in dev that image tag is getting updated with the existing mechanism on the [lines here](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/integration/development/kustomization.yaml#L7-L10).

The change in infra-deployment that will deploy the cronjob is [here](https://github.com/yftacherzog/infra-deployments/commit/cf498e9e6e107905d5dd834ef1b471f1779023ad) (I didn't create a PR yet).